contracts/test/TestJBveBanny.t.sol

```diff
@@ -60,55 +68,131 @@ contract JBveBannyTests is TestBaseWorkflow {
   }
 
   function mintIJBTokens() public returns (IJBToken) {
-    IJBToken _token = _jbTokenStore.tokenOf(_projectId);
+    IJBToken _jbToken = _jbTokenStore.tokenOf(_projectId);
     _projectOwner = projectOwner();
     evm.startPrank(_projectOwner);
     _jbController.mintTokensOf(_projectId, 100 ether, _projectOwner, 'Test Memo', true, true);
-    _token.approve(_projectId, address(_jbveBanny), 10 ether);
-    return _token;
+    _jbToken.approve(_projectId, address(_jbveBanny), 10 ether);
+    return _jbToken;
+  }
+
+  function mintIJBTokensFor(address _account, uint256 _amount) public returns (IJBToken) {
+    IJBToken _jbToken = _jbTokenStore.tokenOf(_projectId);
+    _projectOwner = projectOwner();
+
+    evm.startPrank(_projectOwner);
+    // Mint tokens for project owner
+    _jbController.mintTokensOf(_projectId, _amount * 10, _projectOwner, 'Test Memo', true, true);
+    // Transfer tokens to account
+    _jbToken.transfer(_projectId, _account, _amount);
+    evm.stopPrank();
+
+    // Approve accounts tokens for jbveBanny
+    evm.prank(_account);
+    _jbToken.approve(_projectId, address(_jbveBanny), _amount);
+
+    return _jbToken;
   }
 
   function testLockWithJBToken() public {
     mintIJBTokens();
-    _jbveBanny.lock(_projectOwner, 10 ether, 864000, _projectOwner, true, false);
+    _jbveBanny.lock(_projectOwner, 10 ether, 604800, _projectOwner, true, false);
+    (int128 _amount, , uint256 _duration, bool _useJbToken, bool _allowPublicExtension) = _jbveBanny
+      .locked(1);
+    assert(_jbveBanny.tokenVotingPowerAt(1, block.number) > 0);
```

Comment:
use `assertTrue` and `assertGt` for consistency (both part of DSTest) - same for all occurrences infra